README.md

```diff
@@ -279,17 +279,17 @@ Qpsmtpd plugin: https://github.com/smtpd/qpsmtpd/blob/master/plugins/dmarc
 
 # AUTHORS
 
-- Matt Simerson <msimerson@cpan.org>
-- Davide Migliavacca <shari@cpan.org>
+- Matt Simerson &lt;msimerson@cpan.org>
+- Davide Migliavacca &lt;shari@cpan.org>
 
 # CONTRIBUTORS
 
-- Benny Pedersen <me@junc.eu>
-- Making GitHub Delicious. <iron@waffle.io>
-- Marc Bradshaw <marc@marcbradshaw.net>
-- Priyadi Iman Nurcahyo <priyadi@priyadi.net>
-- Ricardo Signes <rjbs@cpan.org>
-- Ricardo Signes <rjbs@users.noreply.github.com>
+- Benny Pedersen &lt;me@junc.eu>
+- Making GitHub Delicious. &lt;iron@waffle.io>
+- Marc Bradshaw &lt;marc@marcbradshaw.net>
+- Priyadi Iman Nurcahyo &lt;priyadi@priyadi.net>
+- Ricardo Signes &lt;rjbs@cpan.org>
+- Ricardo Signes &lt;rjbs@users.noreply.github.com>
```

Comment:
IIRC, this section is auto-generated by a dist::zilla plugin (and is thus subject to automated reversion). We might look into seeing if that plugin will URL escape for us, and if not, used sed or perl to url encode the angle brackets.